R/linear-mixed-model-deployment.R

```diff
@@ -9,8 +9,8 @@
 #' The linear mixed model functionality works best with data sets having fewer
```

Comment:
It might be nice to have a little bit longer description for deciding when/why to use each type of model. So even in lmm case mentioning that this is for longitudinal data where the data contains multiple rows per patient/observation would be helpful. Though maybe this is most appropriate in each of the development scripts.